pyproject.toml

```diff
@@ -94,9 +98,14 @@ mako = ["mako>=1.2.4"]
 minijinja = ["minijinja>=1.0.0"]
 opentelemetry = ["opentelemetry-instrumentation-asgi"]
 piccolo = ["piccolo"]
-picologging = ["picologging"]
+picologging = ["picologging; python_version < \"3.13\""]
 prometheus = ["prometheus-client"]
-pydantic = ["pydantic", "email-validator", "pydantic-extra-types"]
+pydantic = [
+  "pydantic",
+  "email-validator",
+  "pydantic-extra-types!=2.9.0; python_version < \"3.9\"",
```

Comment:
It looks like only the 2.9.0 version is impacted.  Since it wasn't pulled from pypi, i'm just ignoring this single release.